query/method/message/data/dataCloseRollCall.json

```diff
@@ -0,0 +1,36 @@
+{
+	"$schema": "http://json-schema.org/draft/2019-09/schema#",
+	"$id": "https://github.com/dedis/student20_pop/tree/proto-specs/query/method/message/data/dataCloseRollCall.json",
+	"description": "Match a close roll-call query",
+	"type": "object",
+
+	"properties": {
+		"object": { "const": "roll_call" },
+		"action": { "const": "close" },
+		"id": {
+			"type": "string",
+			"contentEncoding": "base64",
+			"$comment": "Hash : SHA256('R'||lao_id||creation||name)"
```

Comment:
Why include the LAO's ID in the Hash ?

Reply:
(in every files of course)

Reply:
to prevent conflict between multiple identical events from different LAOs, on the same server.
This supports the case of a single server shared across multiple LAOs.